infra/jwt.js

```diff
@@ -1,12 +1,20 @@
 const jwt = require('jsonwebtoken');
-const { jwtSecret } = require('../config/constants');
+const { jwtSecret,jwtRefreshSecret } = require('../config/constants');
 
 exports.createJwt = async obj => {
-	return jwt.sign(obj, jwtSecret, { expiresIn: 86400 * 15 });
+	return jwt.sign(obj, jwtSecret, { expiresIn: 86400 });
+};
+
+exports.createRefreshToken = async obj => {
+	return jwt.sign(obj, jwtRefreshSecret, { expiresIn: 86400 * 7 });
 };
 
 exports.verifyJwt = async token => {
-	return jwt.verify(token, jwtSecret);
+	try{
```

Comment:
- Avoid using try catch blocks for async await calls inside utility functions.
- Instead throw an error in the next() middleware and use a try catch block in the universal error handler

Reply:
@siddinc I had to use a try/catch because verifyJwt should return some value which was then checked in the authRequired function

Reply:
Since this function is highly reusable, let it throw an error. Handle errors where you're using it.